src/sentry/lang/native/symbolicator.py

```diff
@@ -61,10 +61,11 @@
         "itunesUser": {"type": "string", "minLength": 1, "maxLength": 100},
         "itunesCreated": {"type": "string", "format": "date-time"},
         "itunesPassword": {"type": "string"},
-        "itunesSession": {"type": "string"},
         "itunesPersonId": {"type": "string"},
+        "itunesSession": {"type": "string"},
         "appName": {"type": "string", "minLength": 1, "maxLength": 512},
-        "appId": {"type": "string", "minLength": 1, "maxLength": 512},
+        "appId": {"type": "string", "minLength": 1},
+        "bundleId": {"type": "string", "minLength": 1},
```

Comment:
should we also add this property to the required list?

Reply:
good catch.  done